public/style.css

```diff
@@ -29,6 +29,10 @@
 	color: #5d5b6d;
 }
 
+.bg-navbar{
+  color: #9B9B9B;
```

Comment:
I thought it will be more clear since we are using this color only in the navbar background with 10%